packages/mui-material/src/styles/overrides.d.ts

```diff
@@ -111,14 +112,23 @@ import { TooltipClassKey } from '../Tooltip';
 import { TouchRippleClassKey } from '../ButtonBase/TouchRipple';
 import { TypographyClassKey } from '../Typography';
 
-export type OverridesStyleRules<ClassKey extends string = string> = Record<
+export type OverridesStyleRules<
```

Comment:
Let's add some spec tests for the MD components and test that theme augmentation will work.

Reply:
Done!